docs/blog/overrides/pypi-blog.html

```diff
@@ -0,0 +1,30 @@
+{% extends "blog.html" %}
+
+{# A macro to render a blog entry, required. #}
+{% macro render_blog(title, description, time, url, page) -%}
+<a href="{{ page.url }}">
+    <div class="blog-override-post">
+        <h3 class="blog-post-title">{{ title }}</h3>
+        <div class="blog-post-extra">
+            <b>{{ page.meta["author"] + " · " if "author" in page.meta else "" }}</b>
+            <span>{{ page.meta["publish_date"] }}</span>
+        </div>
+        {% if show_tags and "tags" in page.meta %}
+            {% call render_tags(page.meta["tags"], index_url) %}
+            {% endcall %}
+        {% endif %}
+        <p class="blog-post-description">{{ description }}</p>
+    </div>
+</a>
+{{ caller() }}
+{%- endmacro %}
+
+{# Custom css, optional. #}
+{% block style %}
+{{ super() }}
+<style>
+    .md-typeset .blog-post-description {
+        margin-bottom: 0;
+    }
+</style>
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏should this be located in `stylesheets/extra.css` instead?

Reply:
This ended up being vestigial, I removed it.